web/app/css/styles.css

```diff
@@ -1,3 +1,55 @@
+:root {
+  --mdc-theme-primary: #d55a5a;
+  --color-white: #fff;
+}
+
 body {
-  background-color: aliceblue;
+  background: aliceblue url('/images/loginbackground.svg') no-repeat 50% 50%;
```

Comment:
It is better to have all colors as custom properties, so you can move aliceblue there.